mesh/mesquite.cpp

```diff
@@ -847,7 +847,7 @@ static void BoundaryPreservingOptimization(mfem::MesquiteMesh &mesh)
 {
 
    MsqDebug::enable(1);
-   MsqPrintError err(cerr);
+   MsqPrintError err(mfem::err.GetStream());
```

Comment:
Replace `mfem::err.GetStream() -> mfem::err`.

Reply:
Done in Veselin's update.